pkg/hive/hive_test.go

```diff
@@ -175,65 +173,78 @@ func TestBroadcastPeers(t *testing.T) {
 }
 
 func expectOverlaysEventually(t *testing.T, exporter ab.Interface, wantOverlays []swarm.Address) {
+	var (
+		overlays []swarm.Address
+		err      error
+		isIn     = func(a swarm.Address, addrs []swarm.Address) bool {
```

Comment:
Maybe to make this a package level function as it is repeated in two tests. Suggested names `hasAddress`, `containsAddress`...

Reply:
the types are different, one uses `bzz.Address`, the second uses `swarm.Address`

Reply:
Ah, ok. Well, we would have to wait for generics. ;)

Reply:
`interface{}` for the win :trollface: